smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/loader/idl-invalid-list-and-map-members.errors

```diff
@@ -0,0 +1,10 @@
+[ERROR] com.test#NoMemberList: Missing required member of shape `com.test#NoMemberList`: member | Model
+[ERROR] com.test#NoMemberMap: Missing required member of shape `com.test#NoMemberMap`: key | Model
+[ERROR] com.test#MissingKeyMap: Missing required member of shape `com.test#MissingKeyMap`: key | Model
+[ERROR] com.test#MissingValueMap: Missing required member of shape `com.test#MissingValueMap`: value | Model
+[ERROR] com.test#WrongMemberList: Collection shapes may only have a `member` member, but found `foo` | Model
```

Comment:
Rather than `Collection`, can you say the actual shape type name as seen in the IDL / AST?

Reply:
Done